src/modeling/models/mmt_classification_model.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, List
+
+import tensorflow as tf
+from official.modeling import tf_utils
+from official.nlp.keras_nlp import layers
+
+
+class MmtClassificationModel(tf.keras.Model):
+  """Multimodal Transformer-based classification model.
+
+  Args:
```

Comment:
nit: this docstring with the "Args", etc. belongs to the constructor, not to the class, right?

Reply:
Yeah! Edited! Thanks, Santi!